agentMET4FOF/metrological_agents.py

```diff
@@ -106,34 +107,80 @@ def pack_data(self, data, channel="default"):
 
 class MetrologicalMonitorAgent(MetrologicalAgent):
     def init_parameters(self, *args, **kwargs):
-        super().init_parameters(*args, **kwargs)
-
+        super(MetrologicalMonitorAgent, self).init_parameters(*args, **kwargs)
+        #super().init_parameters(*args, **kwargs)
```

Comment:
just one more unused comment here and it should be good to go !

Reply:
Done!